### PR TITLE
FI-1011: Update config.yml documentation

### PR DESCRIPTION
**Pull request by Jammjammjamm**
This branch updates the documentation in config.yml and removes some unused entries.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Code diff has been reviewed for extraneous/missing code
- [ ] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,52 +1,55 @@
-# Name of app
-app_name: Inferno
-
-# Base path of application
+# Base path of application. Inferno will serve pages at HOST/base_path
```
Comment (yunwwang) [inline config.yml:1]: 1) Is there a variable for HOST? or definition of what is HOST?
2) Maybe we could follow FHIR url pattern: [HOST]/[base_path]

File: config.yml [hunk 2]
```diff
@@ -92,7 +95,41 @@ bulk_data_jwks:
...
+# - bulk_public_key: Public key for SMART backend services authorization.
+# - bulk_private_key: Private key for SMART backend services authorization.
+# - bulk_fastest_resource: A FHIR resource type accessible through the patient
+#   and group level exports that the server can export quickly.
+# - bulk_system_export_endpoint: Relative path to system export endpoint from
+#   'bulk_url'.
+# - bulk_patient_export_endpoint: Relative path to patient export endpoint from
+#   'bulk_url'.
+# - bulk_group_export_endpoint: Relative path to group export endpoint from
+#   'bulk_url'.
```
Comment (yunwwang) [inline config.yml:132]: Bulk data program testing uses subset of bulk configuration but it has two additional settings:
- bulk_scope
- group_id

Comment (Jammjammjamm) [reply config.yml:132]: Ok, I added some descriptions. Let me know if they sound reasonable.

Comment (yunwwang) [reply config.yml:132]: The new addition looks good. One more question: Bulkdata test in Program does not use all the community settings. Ex, Program edition tests group export only so bulk_system_export_endpoint, bulk_patient_export_endpoint, and bulk_group_export_endpoint are not used at all. I think it is good to keep the file consistent between program and community. I also think the file should match what settings are actually relevant. So, I will leave the decision to you.

Comment (arscan) [reply config.yml:132]: I think simply sticking to consistency is fine.  I think this does highlight another place where we are putting test-specific stuff in a shared file, which is tough to manage.

**Review by yunwwang — APPROVED** [review]
looks good

**Review by arscan — APPROVED** [review]

**Pull request merged**